src/main/java/org/jabref/logic/importer/NoFetcherFoundException.java

```diff
@@ -0,0 +1,15 @@
+package org.jabref.logic.importer;
+
+import org.jabref.JabRefException;
+
+public class NoFetcherFoundException extends RuntimeException {
```

Comment:
Not sure if this really needs an exception. Normally, you don't want to use an exception to control the flow of the execution and "No fetcher found" doesn't sound like an abnormal situation but more like a special situation in a normal workflow.